BatMobile.java

```diff
@@ -0,0 +1,14 @@
+public class BatMobile extends Car {
+    public void shoot(){

```

Comment:
Нарушение Code Convention:
```suggestion
    public void shoot() {
```